src/PointwiseFunctions/Hydro/Tags.hpp

```diff
@@ -191,6 +192,27 @@ struct SpecificInternalEnergy : db::SimpleTag {
   static std::string name() noexcept { return "SpecificInternalEnergy"; }
 };
 
+/// The vector \f$J^i\f$ in \f$\dot{M} = -\int J^i s_i d^2S\f$,
+/// representing the mass flux through a surface with normal \f$s_i\f$.
```

Comment:
Can you please put a blank line after the first sentence since this paragraph is a very long "brief" description?